ethereum/p2p/src/test/resources/keys/tls_context_factory/compa/avocado/nssdb/pkcs11.txt

```diff
@@ -0,0 +1,5 @@
+library=
+name=NSS Internal PKCS #11 Module
+parameters=configdir='sql:nssdb' certPrefix='' keyPrefix='' secmod='secmod.db' flags= updatedir='' updateCertPrefix='' updateKeyPrefix='' updateid='' updateTokenDescription='' 
+NSS=Flags=internal,critical trustOrder=75 cipherOrder=100 slotParams=(1={slotFlags=[ECC,RSA,DSA,DH,RC2,RC4,DES,RANDOM,SHA1,MD5,MD2,SSL,TLS,AES,Camellia,SEED,SHA256,SHA512] askpw=any timeout=30})
+
```

Comment:
this file -> ethereum/p2p/src/test/resources/keys/tls_context_factory/compa/avocado/nssdb/secmod.db is empty - did you mean to add it?

Reply:
yes ... this is a trick/hack to allow SunPKCS11 to use new "sql" based nss database instead of old legacy one.